pom.xml

```diff
@@ -184,27 +185,27 @@
         <version>${spring.version}</version>
       </dependency>
 
-      <dependency>
-        <groupId>javax.transaction</groupId>
-        <artifactId>javax.transaction-api</artifactId>
-        <version>1.2</version>
-      </dependency>
       <dependency>
         <groupId>javax.interceptor</groupId>
```

Comment:
This should be migrated to jakarta as well:
https://mvnrepository.com/artifact/jakarta.interceptor/jakarta.interceptor-api/1.2.4

Reply:
Done in 6bd41c1deb2a348d4f6dcee2f6e1f48c0376b6d3.